src/ui/FacetRange/FacetRange.ts

```diff
@@ -79,6 +79,7 @@ export class FacetRange extends Facet implements IComponentBindings {
      * > The index cannot automatically generate range values for a `FacetRange` whose [`field`]{@link Facet.options.field} option value references a dynamic field generated by a [query function](https://docs.coveo.com/en/232/). In such a case, you _must_ use the `ranges` option.
      */
     ranges: ComponentOptions.buildJsonOption<IRangeValue[]>(),
+    valueTemplateHelper: ComponentOptions.buildStringOption({ defaultValue: 'number' }),
```

Comment:
Should we document this option?

What is the difference between `valueCaption` and `valueTemplateHelper`?

Reply:
Yes I'll document it. Value caption is not for the format and is a dictionary of terms to replace. It's also accepting the JSON format only, so putting something else like a string will not apply the options. That's why a new option is necessary. The string represents the template helper to use e.g. number, size, date, etc.